util/awsservice/constant.go

```diff
@@ -38,16 +39,41 @@ var (
 )
 
 var (
-	ctx                  = context.Background()
-	awsCfg, _            = config.LoadDefaultConfig(ctx)
-	Ec2Client            = ec2.NewFromConfig(awsCfg)
-	EcsClient            = ecs.NewFromConfig(awsCfg)
-	SsmClient            = ssm.NewFromConfig(awsCfg)
-	ImdsClient           = imds.NewFromConfig(awsCfg)
-	CwmClient            = cloudwatch.NewFromConfig(awsCfg)
-	CwlClient            = cloudwatchlogs.NewFromConfig(awsCfg)
-	DynamodbClient       = dynamodb.NewFromConfig(awsCfg)
-	S3Client             = s3.NewFromConfig(awsCfg)
-	CloudformationClient = cloudformation.NewFromConfig(awsCfg)
-	XrayClient           = xray.NewFromConfig(awsCfg)
+	ctx context.Context
```

Comment:
Why are we changing all of them to pointers?

Reply:
To be able to intialize awsConfig with the right region